modules/pipeline/initialize.go

```diff
@@ -175,6 +176,9 @@ func do() (*httpserver.Server, error) {
 	// 同步 pipeline 表拆分后的 commit 字段和 org_name 字段
 	go pipelineSvc.SyncAfterSplitTable()
 
+	// cache client information
```

Comment:
update comment to declare which client clearly, and add TODO for refactor register.